plugins/woocommerce/client/admin/package.json

```diff
@@ -81,9 +81,11 @@
 		"@wordpress/warning": "wp-6.0",
 		"@xstate/react": "3.2.1",
 		"@xstate5/react": "npm:@xstate/react@4",
+		"classnames": "^2.3.2",
```

Comment:
clsx provides similar functionality to classnames, and we've got a direction to migrate to clsx: pdnLyh-6ic-p2

Reply:
Thank you for pointing that out! I’ve replaced `classnames` with `clsx`.